src/main/scala/ch/openolitor/buchhaltung/BuchhaltungRoutes.scala

```diff
@@ -124,8 +123,9 @@ trait BuchhaltungRoutes extends HttpService with ActorReferences
             requestInstance { request =>
               entity(as[RechnungenContainer]) { cont =>
                 onSuccess(buchhaltungReadRepository.getByIds(rechnungMapping, cont.ids)) { rechnungen =>
-                  val xml = generatePain008(rechnungen)
-                  val stream = new ByteArrayInputStream(xml.getBytes(StandardCharsets.UTF_8))
+                  val xml = generatePain008_001_07(rechnungen)
+                  logger.debug(s"=================================================================== xml = $xml")
+                  //val stream = new ByteArrayInputStream(xml.getBytes(StandardCharsets.UTF_8))
                   //storeToFileStore(RechnungExportDaten, Some("exportFile"), stream, "exportFile.xml") { (id, metadata) =>
                   //  complete("File uploaded")
                   //}
```

Comment:
Is this part of another task?

Reply:
done. Next story add the functionality here